packages/e2e/src/pure/readme.md

```diff
@@ -0,0 +1,5 @@
+# Why this module was added?
+
+Importing `@mudita/pure` package to the project and using it with ts-node (that is used by WDIO) causes unexpected problems (`Error [ERR_REQUIRE_ESM]: Must use import to load ES Module:`), mainly by `p-queue` (it uses ESM modules) . One of the solutions to this problem is to downgrade the lib, which is not the best, the other is to adjust the tsconfig/package.json configuration.
+
+To not waste time, the required code from `@mudita/pure` was just copied to this folder, as it is only few types and one class. If you have time or knowledge, feel free to fix this problem and adjust necessary changes.
```

Comment:
What do you think to create issue in our backlog to fix this workround? 🤔

Reply:
Sure, makes sense. Also linked this document with Jira task in https://github.com/mudita/mudita-center/pull/943/commits/4a4272d336144cff5c017837e28bc8ec5073c270